src/app/pipes/display-decimals.pipe.spec.ts

```diff
@@ -19,15 +19,27 @@ describe('DisplayDecimalsPipe', () => {
         expect(pipe.transform('0.0000010112345')).toEqual('<0.00001');
     });
 
+    it('should display 0.00001 if the amount is 0.00001', function() {
+        expect(pipe.transform('0.00001')).toEqual('0.00001');
+    });
+
     it('should display the amount cutoff at 5 decimal points', function() {
-        expect(pipe.transform('0.0011111111111')).toEqual('0.00111');
+        expect(pipe.transform('0.0011111111111')).toEqual('~0.00111[...]');
+    });
+
+    it('should display the amount cutoff at the passed decimal point', function() {
+        expect(pipe.transform('0.0011111111111', 3)).toEqual('~0.001[...]');
+    });
+
+    it('should display an integer when 0 decimal points passed', function() {
+        expect(pipe.transform('1000.9', 0)).toEqual('~1000[...]');
```

Comment:
nitpick: While it seems to be inconsistent having the `[...]` in this case seems a little bit wrong. I would probably do a ~1000 in the case you specify 0 decimals.